recipes/hubspot-api-client/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "hubspot-api-client" %}
+{% set version = "4.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hubspot-api-client-{{ version }}.tar.gz
+  sha256: 621195df4ce231a62bb39739384536441fef0abdede0134b84914b19d3033985
+
+build:
+  noarch: python
+  script:
+    - echo {{ version }} > {{ SRC_DIR }}/VERSION
```

Comment:
Can you report this upstream? Most likely their sdist is broken and missing that file.

Reply:
Hi @ocefpaf 
I started with this version as I needed version <5 for robotframework-windows, It looked like upstream current version has this fixed and will not need this patch once this version is published and upgraded to version 6.0.
I will verify and report if required.